fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -4575,8 +4575,7 @@ void testDeleteWhereComplexGrouped(Map<String, Index> indexMap, boolean isSynthe
                             .join());
 
             if (isSynthetic) {
-                // TODO: come back to this
-                // Issue a delete where to delete the zero group
+                // deleteWhere does not work for synthetic records currently
```

Comment:
This comment isn't correct, it works for synthetic records, but not the synthetic records that are defined in this test.